packages/web-frontend/src/containers/MapOverlayBar/MapOverlayBar.js

```diff
@@ -129,79 +125,6 @@ const MapOverlayBarComponent = ({
     onSetMaxSelectedOverlays(maxNumOfSelectedOverlays);
   };
 
-  const renderDefaultMapOverlay = () => {
```

Comment:
Felicity suggested we get rid of the default map overlay showing at the top, as it's often confused for representing the currently selected map overlay

Reply:
Good call! `selectDefaultMapOverlayCode` is still used elsewhere, but got rid of the first